src/ring_theory/algebraic.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2019 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import ring_theory.integral_closure
+
+/-!
+# Algebraic elements and algebraic extensions
+
+An element of an R-algebra is algebraic over R if it is the root of a nonzero polynomial.
+The main result in this file proves that transitivity of algebraicity:
+a tower of algebraic field extensions is algebraic.
```

Comment:
You should say that an algebra is algebraic over R if all its elements are. Otherwise, the definition does not make sense. Also, a `that` should be removed or replaced by `the`.

Reply:
Thanks, done.